words/tali.asm

```diff
@@ -813,7 +813,7 @@ _check_hex:
                 lda #16
                 bra _base_changed
 _check_binary:
-                cmp #'%'
+                cmp #'%'                ; binary?
```

Comment:
unrelated comment when I was trying to search and figure out if I could write binary constants